rt_utils/rtstruct_builder.py

```diff
@@ -20,18 +20,18 @@ def create_new(dicom_series_path: str) -> RTStruct:
 
         series_data = image_helper.load_sorted_image_series(dicom_series_path)
         ds = ds_helper.create_rtstruct_dataset(series_data)
-        return RTStruct(series_data, ds)
+        return RTStruct(series_data, ds, warn_only=warn_only)
```

Comment:
Lets remove warn_only here (causing build failures)

Try running pytest in the root directory to see if failures persist

Reply:
Done ,py testraaaa